python/py-avro/Portfile

```diff
@@ -17,18 +16,27 @@ description         Avro is a serialization and RPC framework.
 long_description    ${description}
 
 homepage            https://avro.apache.org/
+master_sites        pypi:a/avro
+distname            avro-${version}
```

Comment:
the `master_sites` and `distname` are the default set by the `python` PG, so please remove these redundant lines

Reply:
Thanks done.

Would be good to find some cycles to update https://github.com/macports/macports-contrib/blob/master/pypi2port/pypi2port.py …

Reply:
that's true indeed... I personally don't use that, but use `upt`, which was part of the GSoC last year.